### PR TITLE
Upgrading to 3.9.9.Final because of security alerts

### PR DESCRIPTION
**Pull request by jimwebber**
Had to change a bunch of license things too because netty changed its description.

### CODE REVIEW

File: community/licensecheck-config/src/main/resources/BSD - seedrandom.js [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
 LICENSE (BSD):
```
Comment (chrisvest) [inline community/licensecheck-config/src/main/resources/BSD - seedrandom.js]: I don't think this is right.

**Comment by chrisvest** [comment]
Also, I don't think we need that `community/lucene-index/schema/index/lucene/1/failure-message` file you are adding.

**Comment by chrisvest** [comment]
And please don't add that `enterprise/core-edge/logs/debug.log` file either.

**Comment by jimwebber** [comment]
Whoops, ta @chrisvest. Fat fingers on the git command line there!

**Comment by systay** [comment]
@jimwebber _bump_ The build is failing and nothing has happened in a really long time. Is this PR still valid?

**Comment by systay** [comment]
Looks like this PR is not live. Re-open if I'm wrong

**Pull request closed without merging**